CONTRIBUTING.md

```diff
@@ -311,31 +312,49 @@ scenarios in this guide.
 
 ### Pull Request Guidelines
 
-See the [pull request guidelines](https://github.com/exercism/docs/blob/main/contributing/pull-request-guidelines.md) in the docs repository.
+See the [pull request guidelines][pull-request-guidelines] in the docs repository.
 
 ### Anatomy of an Exercise
 
-See the [anatomy of an exercise](https://github.com/exercism/docs/tree/main/language-tracks/exercises/anatomy) in the docs repository.
+See the [anatomy of an exercise][exercise-anatomy] in the docs repository.
 
 ### Track configuration file
 
-Each language track has a configuration file called `config.json`. Its structure is defined in [this spec](https://github.com/exercism/docs/blob/main/anatomy/tracks/config-json.md). The `config.json` file's key feature is that it defines which exercises the language track has implemented.
+Each language track has a configuration file called `config.json`. Its structure is defined in [this spec][config-json]. The `config.json` file's key feature is that it defines which exercises the language track has implemented.
 
-The [configlet](https://github.com/exercism/docs/blob/main/anatomy/tracks/configlet/README.md) tool can verify (lint) a track's configuration file.
+The [configlet][configlet] tool can verify (lint) a track's configuration file.
 
 ### Git Basics
-See [Git Basics](https://github.com/exercism/docs/blob/main/contributing/git-basics.md) in [docs](https://github.com/exercism/docs) repository.
+
+See [Git Basics][git-basics] in [docs](https://github.com/exercism/docs) repository.
 
 ### etcetera
 
 TODO: add more sections:
 
-- how to merge something locally (for example when there are conflicts, or
+* how to merge something locally (for example when there are conflicts, or
   if you want to fix a small thing without nagging the contributor about it)
 
-
 ## Improving Consistency By Extracting Shared Test Data
 
-This documentation has moved [to the docs repository](https://github.com/exercism/docs/blob/main/you-can-help/improve-exercise-metadata.md).
+This documentation has moved [to the docs repository][improve-exercise-metadata].
 
 We are maintaining this section, since many open issues link to it.
+
+[contributing]: https://exercism.org/contributing
```

Comment:
I wonder if the rules allow keeping these links closer to where they're used?

Reply:
The rules do allow the inline style of linking.

Is that useful?

Having links using this syntax means:

1. One URL can be referenced multiple places in the text (having a single URL makes it difficult to have it close to multiple places of reference).
2. Much easier to keep URLs up to date (supported by collecting all such URLs in one place within a file).

Reply:
It appears that I did not correctly make the intent of my question understood, but that Erik correctly guessed what I meant anyway and managed to answer my question despite my mistake. Thank you.

I meant to ask whether the references are allowed to be placed anywhere other than the bottom of the file. But the guide says to place them at the bottom. This answers my question.

Reply:
It appears that theoretically at least, markdown allows them anywhere. Inasmuch as the Daring Fireball spec does not restrict location: https://daringfireball.net/projects/markdown/syntax#link .

>  Link definitions can be placed anywhere in your Markdown document. I tend to put them immediately after each paragraph in which they’re used, but if you want, you can put them all at the end of your document, sort of like footnotes.

Convention that I have seen and used to date places them at the bottom of each document. And of course, as Erik points out (thanks Erik) the style guide prefers this also.